notebooks/106-auto-device/106-auto-device.ipynb

```diff
@@ -25,9 +25,8 @@
    "metadata": {},
```

Comment:
line break looks strange, maybe it is way to show diff, because on my machine it looks normal

![Screenshot (11)](https://user-images.githubusercontent.com/68700881/222139419-ff6dd845-5a84-4e5d-b88b-aa3ce15f4a4f.png)

Reply:
You're right. This is a problem with ReviewNB rendering